pkg/ecdsa/key_ethereum.go

```diff
@@ -0,0 +1,18 @@
+//+build !celo
+
+package ecdsa
+
+import (
+	cecdsa "crypto/ecdsa"
+
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+// PublicKey holds a public key of the Signer.
+type PublicKey cecdsa.PublicKey
+
+// Marshal serializes Public Key to bytes in uncompressed form as described in
+// [SEC 1] section 2.3.3: `04 + <x coordinate> + <y coordinate>`
+func (pk *PublicKey) Marshal() []byte {
+	return crypto.FromECDSAPub((*cecdsa.PublicKey)(pk))
+}
```

Comment:
Is this function used anywhere? I commented it out and I was able to build the client. Maybe we can avoid having two versions of this file.

If this function is no longer used I'd also love to get rid of our own `ecdsa.PublicKey` and use one from `crypto/ecdsa`.

Reply:
Looks you're right but this can be used for implicit marshaling somewhere. I'll make sure we are good with removing them and if so I'll do it.